index_settings_test.go

```diff
@@ -378,6 +380,47 @@ func TestIndex_GetSortableAttributes(t *testing.T) {
 	}
 }
 
+func TestIndex_GetTypoTolerance(t *testing.T) {
+	type args struct {
+		UID    string
+		client *Client
+	}
+	tests := []struct {
```

Comment:
This particular struct is different in every test? I was thinking of a way to make it reusable.

Reply:
The tests were autogenerated with `GoLand` but this soft is not free and I just use it for this purpose. I think It could be nice to find a free solution and add it to the `CONTRIBUTING`.